feature/notice_detail/src/main/java/com/ku_stacks/ku_ring/notice_detail/component/PreviewData.kt

```diff
@@ -0,0 +1,23 @@
+package com.ku_stacks.ku_ring.notice_detail.component
+
+import com.ku_stacks.ku_ring.domain.NoticeComment
+import com.ku_stacks.ku_ring.domain.PlainNoticeComment
+
+internal val fakePlainComment = PlainNoticeComment(
+    id = 0,
+    parentNoticeId = 0,
+    authorId = 0,
+    authorName = "쿠링",
+    noticeId = 0,
+    content = "쿠링 댓글 내용".repeat(10),
+    postedDatetime = "2025.03.23 20:27",
+    updatedDatetime = "2025.03.23 20:27",
+)
+internal const val size = 10
+internal val fakeComments = List(size) { index ->
+    NoticeComment(
+        comment = fakePlainComment.copy(id = index, authorName = "쿠링 $index"),
+        replies = List(2) { fakePlainComment.copy(id = index * 10 + it) },
+        hasNext = (index == size - 1),
+    )
+}
```

Comment:
```suggestion
internal val fakeComments: (Int) -> List<NoticeComment> = { size -> 
    List(size) { index ->
        NoticeComment(
            comment = fakePlainComment.copy(id = index, authorName = "쿠링 $index"),
            replies = List(2) { fakePlainComment.copy(id = index * 10 + it) },
            hasNext = (index == size - 1),
        )
    }
}
```

이런 방식은 어떨까요 그러면 size를 선언부에서도 사용할 수 있어서

Reply:
이런 리뷰는 처음인데, 생각해보니 always better네요. https://github.com/ku-ring/KU-Ring-Android/pull/451/commits/7232666e2f1aa1541f367913c596ec6c8753e700 에서 수정했습니다